src/repositories/api/realm/api-realm-repository-impl.ts

```diff
@@ -79,4 +85,27 @@ export class ApiRealmRepositoryImpl implements ApiRealmRepository {
         return result.data?.data;
       });
   }
+
+  async getRealmStorageDeposit(realmPath: string): Promise<StorageDeposit | null> {
+    if (!this.nodeClient) {
+      throw new CommonError("FAILED_INITIALIZE_PROVIDER", "NodeRPCClient");
+    }
+
+    const response = await this.nodeClient.abciQueryVMStorageDeposit(realmPath).catch(() => null);
+    if (!response || !response?.response?.ResponseBase?.Data) {
+      return null;
+    }
+
+    try {
+      const rawResult = parseABCIKeyValueResponse(response.response.ResponseBase.Data);
+
+      if (hasStorageDepositProperties(rawResult)) {
+        return convertToStorageDeposit(rawResult);
+      }
+
+      return null;
+    } catch (e) {
+      return null;
+    }
```

Comment:
we cannot identify the problem even when an error occurs. How about adding logs?